Source/Blazorise/wwwroot/breakpoint.js

```diff
@@ -2,15 +2,17 @@
 const breakpointComponents = [];
 let lastBreakpoint = null;
 
-// Recalculate breakpoint on resize
-if (window.attachEvent) {
-    window.attachEvent('onresize', windowResized);
-}
-else if (window.addEventListener) {
-    window.addEventListener('resize', windowResized, true);
-}
-else {
-    //The browser does not support Javascript event binding
+if (document.readyState === 'interactive' || document.readyState === 'complete') {
+    // Recalculate breakpoint on resize
+    if (window.attachEvent) {
+        window.attachEvent('onresize', windowResized);
+    }
+    else if (window.addEventListener) {
+        window.addEventListener('resize', windowResized, true);
```

Comment:
Are the events guaranteed to be only registered once?
Also I dont see them being disposed? Is it needed?